### DISCUSSION TITLE
I have Dev mode enabled after I esc-reload-power? (You could also test this if you want)

### DISCUSSION THREAD

**Discussion by Luigi-Master** [3kh0/ext-remover#1222] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I'm wondering if its just my imagination but I did control d on recovery screen and the developer mode screen popped up, I pressed confirmed, it powerwashed my Chromebook as normal, and then I put my school email and that's when it enrolled me into the district. (also I did this at home using my home network,) SOOO, what if I put my personal Gmail instead, Will it not allow or will I just be able to bypass the restrictions. Thank you for your time!

**Comment by CGRmags** [comment]
shut up

On Tue, Oct 29, 2024 at 4:16 AM Luigi-Master ***@***.***>
wrote:

> I'm wondering if its just my imagination but I did control d on recovery
> screen and the developer mode screen popped up, I pressed confirmed, it
> powerwashed my Chromebook as normal, and then I put my school email and
> that's when it enrolled me into the district. (also I did this at home
> using my home network,) SOOO, what if I put my personal Gmail instead, Will
> it not allow or will I just be able to bypass the restrictions. Thank you
> for your time!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/1222>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBFFTZLEOEQ6LYEZ6IDZ55U7ZAVCNFSM6AAAAABQZTECQ2VHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM4TAMBTGE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Reply by Luigi-Master** [reply]
is that a joke, or is that rude

**Reply by Sam-hacks1** [reply]
it doesn't allow

**Reply by Luigi-Master** [reply]
sorry then its blocked for you, for some people like me its not blocked

**Reply by Sam-hacks1** [reply]
when i tried it said that I couldn't log in with a not school email

**Reply by Luigi-Master** [reply]
did you do this at home or at school

**Reply by Sam-hacks1** [reply]
home

**Reply by Luigi-Master** [reply]
yeah they patched it but do you have an Android phone

**Reply by Sam-hacks1** [reply]
ye

**Reply by Luigi-Master** [reply, marked as answer]
ok right when you power wash your Chromebook and click us language, I think that the wifi screen should show up or any pop up with a qr code, scan that with your android phone but remember to have your personal Gmail synced with it, then after you scan do the next steps and the Chromebook should be your personal know unless they patched it.
